DeepSpeech.py

```diff
@@ -3,179 +3,168 @@
 from __future__ import absolute_import
 # -*- coding: utf-8 -*-
 
+import os
+
+if 'TF_CPP_MIN_LOG_LEVEL' not in os.environ:
+    os.environ['TF_CPP_MIN_LOG_LEVEL'] = '3'
```

Comment:
Is log level 3 what we actually want here? I'd like to know about errors at least by default, maybe even warnings.